spring-cloud-gcp-starters/spring-cloud-gcp-starter-firestore/pom.xml

```diff
@@ -9,7 +9,7 @@
 		<version>2.0.0-RC1-SNAPSHOT</version>
 	</parent>
 	<artifactId>spring-cloud-gcp-starter-firestore</artifactId>
-	<name>Spring Cloud GCP Cloud Firestore Starter</name>
+	<name>Spring Cloud GCP Starter - Cloud Firestore</name>
```

Comment:
I'm on the fence about prepending `Cloud` -- if we do for `Cloud Firestore`, we should for `Cloud Spanner`, but when appended to `Spring Cloud GCP` it's just soooo verbose.

Reply:
Changed just to Firestore.